cmd/soroban-rpc/internal/test/integration.go

```diff
@@ -96,11 +98,36 @@ func NewTest(t *testing.T) *Test {
 	return i
 }
 
+func (i *Test) waitForCheckpoint() {
+	i.t.Log("Waiting for core to be up...")
```

Comment:
Most probably unrelated to this PR, but how can integration tests work without waiting for the checkpoint to be reached? Without it, the DB will be empty.

Reply:
Ahhhh